cognite/powerops/custom_modules/power_model_v1/data_models/views/bid_configuration/1-interface.BidConfiguration.view.yaml

```diff
@@ -29,9 +29,9 @@ properties:
   marketConfiguration:
     container:
       space: '{{powerops_models}}'
-      externalId: FunctionData
+      externalId: BidConfiguration
       type: container
-    containerPropertyIdentifier: direct2
```

Comment:
This is causing all views that implements this one to become non-writable (not detected by FDM due to bug)